### PR TITLE
This is a mashup PR that contains all the latest UI changes/fixes that have been done lately

### PR DESCRIPTION
**Pull request by valpinkman**
To make it easier to QA, all the previous PR have been reviewed by devs, and just need a QA pass

- https://github.com/LedgerHQ/ledger-live-desktop/pull/3387 => [LL-3914](https://ledgerhq.atlassian.net/browse/LL-3914)
- https://github.com/LedgerHQ/ledger-live-desktop/pull/3393 => [LL-3961](https://ledgerhq.atlassian.net/browse/LL-3961)
- https://github.com/LedgerHQ/ledger-live-desktop/pull/3402 => [LL-4170](https://ledgerhq.atlassian.net/browse/LL-4170)
- https://github.com/LedgerHQ/ledger-live-desktop/pull/3414 => [LL-4258](https://ledgerhq.atlassian.net/browse/LL-4258)
- https://github.com/LedgerHQ/ledger-live-desktop/pull/3416 => [LL-2803](https://ledgerhq.atlassian.net/browse/LL-2803)
- https://github.com/LedgerHQ/ledger-live-desktop/pull/3423 => [LL-4185](https://ledgerhq.atlassian.net/browse/LL-4185)
- https://github.com/LedgerHQ/ledger-live-desktop/pull/3424 => [LL-4261](https://ledgerhq.atlassian.net/browse/LL-4261)

### Type

Various UI Fixes

### CODE REVIEW

File: src/renderer/components/TabBar.js [hunk 1]
```diff
@@ -114,7 +112,7 @@ const TabBar = ({
           active={j === i}
           tabIndex={j}
           onClick={() => updateIndex(j)}
-          id={withId ? `settings-${tab}-tab` : ""}
+          id={withId ? `settings-${tab.toLowerCase()}-tab` : ""}
```
Comment (machard) [inline src/renderer/components/TabBar.js:115]: i believe ids only work on native elements

**Review by gre — APPROVED** [review]

**Review by nabil-brn — CHANGES_REQUESTED** [review]
- [x] #3387 => LL-3914 -> no more present in algorand earn reward modal, only jar?
- [x] #3393 => LL-3961 -> navbar font is too big compared to figma
- [ ] #3402 => LL-4170 -> icons look stretched
- [x] #3414 => LL-4258
- [x] #3416 => LL-2803 
- [x] #3423 => LL-4185 -> "NEVER" is not in bold, and the second point of jira ticket is not fixed
- [ ] #3424 => LL-4261 -> from CTA (or not) when I've system theme (dark), the onboarding theme is dusk

---
06088200b (HEAD -> pr/valpinkman/3425, val/ui/mashup-pr) Update TabBar.js

**Comment by gre** [comment]
you are close @valpinkman  =D

**Review by nabil-brn — CHANGES_REQUESTED** [review]
- [x]  #3387 => LL-3914
- [x]  #3424 => LL-4261
- [x]  #3402 => LL-4170
- [x]  #3414 => LL-4258
- [x]  #3416 => LL-2803
- [ ]  #3393 => LL-3961 -> Sub section font in settings are too small compared to figma
image
- [ ]  #3423 => LL-4185
- NEVER / ALWAYS should be bold
image
- PIN CODE step (icon should be a cross)
image

---
42e17cc03 (HEAD -> pr/valpinkman/3425) update algorand rewards ilustration

**Review by nabil-brn — CHANGES_REQUESTED** [review]
Icon is fixed but now I have a mix of languages. I though it was fixed on develop already ?
image

NEVER / ALWAYS should be in **bold**.
image

---
b848b4c0d (grafted, HEAD -> pr/valpinkman/3425, val/ui/mashup-pr) fix eslint

**Review by nabil-brn — DISMISSED** [review]

**Review by nabil-brn — APPROVED** [review]

**Pull request merged**